src/index.js

```diff
@@ -39,9 +40,12 @@ export default function supernova(env) {
       const constraints = useConstraints();
       const translator = useTranslator();
       const selectionsAPI = useSelections();
-      const tableTheme = muiSetup(direction);
-      // the style which is from them can be found in nebula.config.js -> serve -> themes
-      const theme = useTheme();
+      const theme = {
+        ...useTheme(),
+        backgroundColor: useTheme().getStyle('object', 'straightTable', 'backgroundColor'),
```

Comment:
So this would override if you have a general backgroundColor or?

Reply:
so here what it does is just add two new properties (backgroundColor, isBackgroundDarkColor) into the object from useTheme(). It does not override anything.

Reply:
ahhh, yeah there is no general backgroundColor, this is a new prop 👍